test/spec/token.js

```diff
@@ -8,7 +8,7 @@ define(function(require) {
   var Q = require('q');
 
   function setupSync() {
-    return new OktaAuth({ url: 'http://example.okta.com' });
+    return new OktaAuth({ issuer: 'http://example.okta.com' });
```

Comment:
Do we still have a test elsewhere that asserts that `url` is still a valid option to pass when building a client?

Reply:
Yep - its used throughout our test suite.

While this PR changes `url` -> `issuer` in our OIDC flow tests, we're keeping all non-OIDC flows unchanged.